.storybook/custom/custom.scss

```diff
@@ -440,3 +440,13 @@ body.sb-show-main {
 .fddocs-select {
   margin-right: 15px;
 }
+
+.mobile-container {
```

Comment:
can we rename the class with a prefix `fddocs-`

Reply:
isn't `fddocs` prefix supposed to be applied if we are overriding `fd-*` elements? I am not overriding any fd element here, so didn't add it
https://github.com/SAP/fundamental-styles/pull/1752#discussion_r500546152